### PR TITLE
Alarm/Alert/Info duration

### PR DESCRIPTION
**Pull request by Snsei987**
#591

### CODE REVIEW

File: ui/raidboss/popup-text.js [hunk 1]
```diff
@@ -292,7 +292,15 @@ class PopupText {
 
     let userDisabled = trigger.id && this.options.DisabledTriggers[trigger.id];
     let delay = 'delaySeconds' in trigger ? ValueOrFunction(trigger.delaySeconds) : 0;
-    let duration = 'durationSeconds' in trigger ? ValueOrFunction(trigger.durationSeconds) : 3;
+    let duration = {
+      fromTrigger:
+        'durationSeconds' in trigger
+          ? ValueOrFunction(trigger.durationSeconds)
+          : undefined,
```
Comment (quisquous) [inline ui/raidboss/popup-text.js]: Should undefined here be 3 as the default?

File: ui/raidboss/popup-text.js [hunk 2]
```diff
@@ -355,7 +363,10 @@ class PopupText {
           let holder = that.alarmText.getElementsByClassName('holder')[0];
           let div = makeTextElement(text, 'alarm-text');
           addText.bind(that)(holder, div);
-          window.setTimeout(removeText.bind(that, holder, div), duration * 1000);
+          window.setTimeout(
+              removeText.bind(that, holder, div),
+              (duration.fromTrigger || duration.alarmText) * 1000
```
Comment (quisquous) [inline ui/raidboss/popup-text.js]: Can this be max of duration.alarmText and duration.fromTrigger? If somebody specifies a really long alarmText option, a specified duration on the trigger should not make it shorter, I think?

Comment (Snsei987) [reply ui/raidboss/popup-text.js]: Well my idea was as no durationSecond is specified by default, the user had to add it manually therefore he wants it to be the time for this specific trigger.

**Comment by quisquous** [comment]
Thanks for this!!

**Pull request merged**